.gitignore

```diff
@@ -5,3 +5,4 @@
 node_modules
 test-results
 .jest-cache
+.tgz
```

Comment:
```suggestion
*.tgz
```

Reply:
thanks Matt didn't realize i needed to glob that